apps/obs-web/src/views/DebrieferErrors.vue

```diff
@@ -3,79 +3,133 @@
-import {
-  WcgopTrip,
-  WcgopOperation,
-  WcgopCatch,
-  WcgopSpecimen,
-  Basket
-} from '@boatnet/bn-models';
-import { CouchDBCredentials, couchService } from '@boatnet/bn-couch';
-import { Client, CouchDoc, ListOptions } from 'davenport';
-import { date } from 'quasar';
-import { convertToObject } from 'typescript';
-
-@Component
-export default class DebrieferErrors extends Vue {
-  @Action('error', { namespace: 'alert' }) private error: any;
-  @State('debriefer') private debriefer!: DebrieferState;
-  @Prop({ default: true })
-  private showData!: boolean;
-
-  private WcgopTrips: WcgopTrip[] = [];
-  private WcgopOperations: WcgopOperation[] = [];
-  private WcgopCatches: WcgopCatch[] = [];
-  private WcgopCatchSpecies: WcgopCatch[] = [];
-  private WcgopCatchBaskets: Basket[] = [];
-  private WcgopCatchSpecimens: WcgopSpecimen[] = [];
-  private pagination = { rowsPerPage: 50 };
-  private tab = 'trips';
-
-  private errorColumns = [
-    { field: 'severity', header: 'Severity' },
-    { field: 'description', header: 'Description' },
-    { field: 'tripNum', header: 'Trip #' },
-    { field: 'dateCreated', header: 'Date Created' },
-    { field: 'observer', header: 'Observer' },
-    { field: 'status', header: 'Status' },
-    { field: 'dateFixed', header: 'Date Fixed' },
-    { field: 'note', header: 'Note' }
-  ];
-
-  private data = [
-    {
-      severity: 'Warning',
-      description:
-        'Multiple dissections of the same type collected for sea whip, sea pen, or non-coral species.',
-      tripNum: 24266,
-      dateCreated: '1/1/19',
-      observer: 'Davis',
-      status: 'Unresolved',
-      dateFixed: '',
-      note: 'dissection count'
-    }
-  ];
-
-  private openNewDebriefingTab() {
-    const route = '/observer-web/debriefer/qa';
-    window.open(route, '_blank');
+import { createComponent, ref, onMounted, watch, computed } from '@vue/composition-api';
+import moment from 'moment';
+
+import { couchService } from '@boatnet/bn-couch';
+import { Client } from 'davenport';
+import { NoSubstitutionTemplateLiteral } from 'typescript';
+
+export default createComponent({
+  props: {
+    showData: Boolean,
+    isFullSize: Boolean
+  },
+  setup(props, context) {
+    const store = context.root.$store;
+    const state = store.state;
+    const masterDB: Client<any> = couchService.masterDB;
+
+    const errorColumns = [
+      { field: 'severity', header: 'Severity', key: 'errorSeverity', width: '80' },
+      { field: 'description', header: 'Description', key: 'errorDescription', width: '80'},
+      { field: 'tripNum', header: 'Trip #', key: 'errorTripNum', width: '80' },
+      { field: 'dateCreated', header: 'Date Created', key: 'dateErrorCreated', width: '80' },
+      { field: 'observer', header: 'Observer', key: 'errorObserver', width: '80' },
+      { field: 'status', header: 'Status', type: 'toggle',
+                listType: 'template',
+                list: ['Unresolved', 'Resolved'],
```

Comment:
Based off these notes [here](https://docs.google.com/document/d/10TcFfWZ49O96NKLa9FcBF5lwG0rlN8wbIKU4-V7X_I0/edit#) looks like the statuses they want are: Resolved, valid, pending, ignore, and flag. Can discuss this more in the upcoming meeting. Also just curious do you think we should store theses as lookups? 

Here's some of the notes from the doc. Might be useful for other things. 
TERs
Auto-generated errors from rules engine
Status - Resolved, Valid, Pending, Ignore, Flag
Debriefer Notes - free text
Filtering
Severity - warnings v. errors
Status -  
Free text filtering - across all columns